backend/src/main/java/com/daedan/festabook/storage/infrastructure/LocalStorageManager.java

```diff
@@ -0,0 +1,65 @@
+package com.daedan.festabook.storage.infrastructure;
+
+import com.daedan.festabook.storage.domain.StorageManager;
+import com.daedan.festabook.storage.dto.StorageUploadRequest;
+import com.daedan.festabook.storage.dto.StorageUploadResponse;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@Profile("dev")
+public class LocalStorageManager implements StorageManager {
+
+    private final String basePath;
+
+    public LocalStorageManager(
+            @Value("${local.storage.base-path}") String basePath
+    ) {
+        this.basePath = basePath;
+    }
```

Comment:
```suggestion
    @Value("${local.storage.base-path}")
    private final String basePath;
```

전부터 느낀 거지만... 이 방식과 비교했을 때 어떤 차이점이 있나요?